client/app/helpers/optionset.js

```diff
@@ -54,40 +68,34 @@ const OPTIONSET_LOOKUP = {
  * @return     {string, number, array or Object}
  */
 export function optionset([model, optionsetId, returnType, lookupToken]) {
-  const optionset = OPTIONSET_LOOKUP[model][optionsetId];
+  const optionset = OPTIONSET_LOOKUP[model][optionsetId]; // eslint-disable-line
+  const optionById = optionset[lookupToken];
+  let option;
 
   switch (returnType) {
     case 'list':
       return Object.values(optionset);
     case 'code':
-      if (lookupToken) {
-        const optionById = optionset[lookupToken];
-
-        if (optionById) {
-          return optionById.code;
-        }
+      if (optionById) {
+        return optionById.code;
+      }
 
```

Comment:
I really don't want to change this stuff right now. I'm not confident we're asserting that lookups are working correctly anyway.

Please use ember-composable-helpers and build on top of the existing code, or add your own helper if this doesn't do the job.

Reply:
I paired on this with @godfreyyeung and we manually checked the other places where this helper is used to make sure if still functions as intended. This change allows the `optionset` helper to look up falsey values. In refactoring this helper, we realized that this `if()` was never actually necessary. If you want to keep it this helper unchanged, I'll need to back out all the changes that make use of it—which is fine with me, it just makes the template code more verbose.

Reply:
Alright, if ya'll checked how it'd affect the other parts of the code, I think that's fine.

Thanks andy!